core/src/validator.rs

```diff
@@ -913,6 +914,11 @@ impl Validator {
         let (snapshot_request_sender, snapshot_request_receiver) = unbounded();
         let accounts_background_request_sender =
             AbsRequestSender::new(snapshot_request_sender.clone());
+        let snapshot_controller = Arc::new(SnapshotController::new(
+            accounts_background_request_sender,
+            Some(config.snapshot_config.clone()),
+            bank_forks.read().unwrap().root(),
```

Comment:
I don't think it really matters what the latest request slot is initialized to actually, but I used the bank forks root to match the previous implementation